doc/src/development/testenv/theo-setup-vm-windows.md

```diff
@@ -16,7 +16,7 @@ You may also install the required software manually.
     choco install git.install --params "'/GitAndUnixToolsOnPath /WindowsTerminal /NoAutoCrlf'"
     ```
 
-> **Warning**
+> **Warning**  
```

Comment:
There sneaked in some spaces.

Reply:
The 2 spaces were introduced to have a line break after "Warning".